api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/MedicationRequestDispenseRequestComponentTranslatorImpl.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators.impl;
+
+import javax.annotation.Nonnull;
+
+import lombok.AccessLevel;
+import lombok.Setter;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.MedicationRequest;
+import org.hl7.fhir.r4.model.Quantity;
+import org.openmrs.Concept;
+import org.openmrs.DrugOrder;
+import org.openmrs.module.fhir2.api.translators.ConceptTranslator;
+import org.openmrs.module.fhir2.api.translators.MedicationRequestDispenseRequestComponentTranslator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@Setter(AccessLevel.PACKAGE)
+public class MedicationRequestDispenseRequestComponentTranslatorImpl implements MedicationRequestDispenseRequestComponentTranslator {
+	
+	@Autowired
+	private ConceptTranslator conceptTranslator;
+	
+	@Override
+	public MedicationRequest.MedicationRequestDispenseRequestComponent toFhirResource(@Nonnull DrugOrder drugOrder) {
+		MedicationRequest.MedicationRequestDispenseRequestComponent dispenseRequestComponent = new MedicationRequest.MedicationRequestDispenseRequestComponent();
+		if (drugOrder.getQuantity() != null) {
+			Quantity quantity = new Quantity();
+			quantity.setValue(drugOrder.getQuantity());
+			if (drugOrder.getQuantityUnits() != null) {
+				CodeableConcept units = conceptTranslator.toFhirResource(drugOrder.getQuantityUnits());
+				Coding coding = units.getCodingFirstRep();
```

Comment:
I assume you've researched/thought about this, but how is the "first rep" determined?
Do we want to consider using the the same strategy we use here for DosageUnits (using RX_Norm if found, then SNOMED, otherwise just uuid)?

Reply:
I've added a new commit to address this @mogoodrich .  Please have a look.